### PR TITLE
Browser `tracingOrigins` is deprecated and replaced with `tracePropagationTargets`

### PR DESCRIPTION
**Pull request by timfish**
This PR replaces `tracingOrigins` with `tracePropagationTargets`. 

I have not touched `tracingOrigins` in relation to these because they will need to updated to the latest JavaScript SDK for these changes to become relevant:
- React Native
- Electron `Net` integration
- Wizard docs for capacitor/ionic

It also updated the description for `shouldCreateSpanForRequest` because it no longer considers `tracingOrigins`.

### CODE REVIEW

File: src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 1]
```diff
@@ -50,15 +50,15 @@ Sentry.init({
 
 Supported options:
 
-### tracingOrigins
+### tracePropagationTargets
```
Comment (AbhiPrasad) [inline src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx:53]: Can we add a note saying that this was previously called `tracingOrigins`?

File: src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 2]
```diff
@@ -50,15 +50,17 @@ Sentry.init({
...
+
+Previously called `tracingOrigins`.
```
Comment (AbhiPrasad) [inline src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx:55]:
```suggestion
<Note>

The `tracingOrigins` option was was renamed to `tracePropagationTargets` and deprecated in [version `7.19.0`](https://github.com/getsentry/sentry-javascript/releases/tag/7.19.0) of the JavaScript SDK. `tracingOrigins` will be removed in the v8 version.

</Note>
```

File: src/platform-includes/performance/connect-services/javascript.mdx [hunk 3]
```diff
@@ -10,7 +10,7 @@ The `baggage` header was added with version 7 of the Sentry Javascript SDK and i
 
 All of Sentry's tracing-related integrations (`BrowserTracing`, `Http`, and `Express`), as well as the Next.JS SDK, either generate or pick up and propagate the trace headers automatically, as appropriate, for all transactions and spans that they generate.
 
-The JavaScript SDK will only attach the trace headers to outgoing HTTP requests for which the destination is a substring or regex match to the <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#tracingorigins">tracingOrigins</PlatformLink> list.
+The JavaScript SDK will only attach the trace headers to outgoing HTTP requests for which the destination is a substring or regex match to the <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#tracePropagationTargets">tracePropagationTargets</PlatformLink> list.
```
Comment (AbhiPrasad) [inline src/platform-includes/performance/connect-services/javascript.mdx:13]:
```suggestion
The JavaScript SDK will only attach the trace headers to outgoing HTTP requests for which the destination is a substring or regex match to the <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#tracePropagationTargets">tracePropagationTargets</PlatformLink> list. `tracePropagationTargets` was previously called `tracingOrigins`.
```

**Review by AbhiPrasad — APPROVED** [review]

File: src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 4]
```diff
@@ -50,15 +50,21 @@ Sentry.init({
 
 Supported options:
 
-### tracingOrigins
+### tracePropagationTargets
+
+<Note>
+
+The `tracingOrigins` option was was renamed to `tracePropagationTargets` and deprecated in [version `7.19.0`](https://github.com/getsentry/sentry-javascript/releases/tag/7.19.0) of the JavaScript SDK. `tracingOrigins` will be removed in the v8 version.
```
Comment (lizokm) [inline src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx:57]:
```suggestion
The `tracingOrigins` option was renamed `tracePropagationTargets` and deprecated in [version `7.19.0`](https://github.com/getsentry/sentry-javascript/releases/tag/7.19.0) of the JavaScript SDK. `tracingOrigins` will be removed in version 8.
```

File: src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 5]
```diff
@@ -50,15 +50,21 @@ Sentry.init({
...
+
+</Note>
 
 A list of strings and regular expressions. The JavaScript SDK will attach the `sentry-trace` and `baggage` headers to all outgoing XHR/fetch requests whose destination contains a string in the list or matches a regex in the list. If your frontend is making requests to a different domain, you'll need to add it there to propagate the `sentry-trace` and `baggage` headers to the backend services, which is required to link transactions together as part of a single trace.
 
-**The `tracingOrigins` option matches against the whole request URL, not just the domain. Using stricter regex to match certain parts of the URL ensures that requests do not unnecessarily have the additional headers attached.**
+**The `tracePropagationTargets` option matches against the whole request URL, not just the domain. Using stricter regex to match certain parts of the URL ensures that requests do not unnecessarily have the additional headers attached.**
```
Comment (lizokm) [inline src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx:63]:
```suggestion
**The `tracePropagationTargets` option matches the entire request URL, not just the domain. Using stricter regex to match certain parts of the URL ensures that requests don't unnecessarily have additional headers attached.**
```

File: src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 6]
```diff
@@ -70,7 +76,9 @@ You will need to configure your web server [CORS](https://developer.mozilla.org/
 
 ### shouldCreateSpanForRequest
 
-This function can be used to filter out unwanted spans such as XHR's running health checks or something similar. Whether specified or not, `shouldCreateSpanForRequest` filters out everything but what was defined in `tracingOrigins`.
+This function can be used to filter out unwanted spans such as XHR's running
+health checks or something similar. If this function is not specified, spans
+will be created for all requests.
```
Comment (lizokm) [inline src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx:81]:
```suggestion
This function can be used to filter out unwanted spans such as XHRs running
health checks or something similar. If this function isn't specified, spans
will be created for all requests.
```

**Review by lizokm — COMMENTED** [review]
Made some tweaks to the language.

**Pull request merged**